pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -189,6 +190,10 @@ public BrokerService(PulsarService pulsar) throws Exception {
 
         this.multiLayerTopicsMap = new ConcurrentOpenHashMap<>();
         this.pulsarStats = new PulsarStats(pulsar);
+        // register listener to capture zk-latency
+        ClientCnxnAspect.addListner((eventType, latencyMs) -> {
+            this.pulsarStats.recordZkLatencyTimeValue(eventType, latencyMs);
```

Comment:
Actually, right now, we are using BK-Yahoo version where [BK-ZK-Client](https://github.com/yahoo/bookkeeper/blob/yahoo-4.3/bookkeeper-server/src/main/java/org/apache/bookkeeper/zookeeper/ZooWorker.java) captures startTime in Msec and it doesn't have OpStatsLoggger yet. And it seems both the things are fixed in [BK-master](https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/zookeeper/ZooWorker.java). 
Therefore, startTime is milliseconds, it's not possible to capture time < 1 msec for now. We can do it once we move to latest BK version.